build.gradle

```diff
@@ -0,0 +1,37 @@
+plugins {
+    id 'java'
+    id 'maven-publish'
```

Comment:
it looks like it should be removed to avoid publishing builders module artifact to maven local repo:

`id 'maven-publish'`